Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /app
 
 RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
 RUN echo "deb http://repo.mongodb.org/apt/debian jessie/mongodb-org/3.6 main" > /etc/apt/sources.list.d/mongodb-org-3.6.list
-RUN apt-get update
+RUN sed -i '/jessie-updates/d' /etc/apt/sources.list && apt-get update
```

Comment:
arf I am not a big fan of the `sed` solution. Can we change the base image? and use `node:8.12.0-stretch` instead ?

Reply:
We should be able to, I’ll try to resolve the `libssl` issue.

Reply:
:/ :notlikethis: